src/main/java/com/celements/navigation/Navigation.java

```diff
@@ -77,13 +81,12 @@
 
   private INavigationBuilder navBuilder;
 
-  public static final String LIST_LAYOUT_TYPE = "list";
-
-  private static final String _PAGE_MENU_DATA_TYPE = "pageMenu";
+  @Deprecated
+  public static final String LIST_LAYOUT_TYPE = NavigationConfig.LIST_LAYOUT_TYPE;
 
   public static final String MENU_TYPE_MENUITEM = "menuitem";
```

Comment:
Is there a reason MENU_TYPE_MENUITEM was not moved to NavigationConfig like all the other constants?